spec/genre_spec.rb

```diff
@@ -0,0 +1,13 @@
+require './genre'
+require './item'
+
+describe Genre do
+  context 'when initialized' do
+    it 'Should create a genre' do
+      genre = Genre.new('Ali')
+      item = Item.new('2022-10-01')
```

Comment:
Your Genre gets associated with a MusicAlbum object, so kindly use a MusicAlbum instance object instead of an Item instance object.